README.md

```diff
@@ -5,4 +5,65 @@
 
 Ariadne is a Python library for implementing [GraphQL](http://graphql.github.io/) servers, inspired by [Apollo Server](http://graphql.github.io/) and built with [GraphQL-core](https://github.com/graphql-python/graphql-core).
 
-**Warning**: The work on library is currently in experimental phase. We invite you to give Ariadne a try, but if you are looking to build production-ready GraphQL API, please use more stable solutions such as [Graphene](https://github.com/graphql-python/graphene).
\ No newline at end of file
+**Warning**: The work on library is currently in experimental phase. We invite you to give Ariadne a try, but if you are looking to build production-ready GraphQL API, please use more stable solutions such as [Graphene](https://github.com/graphql-python/graphene).
+
+## Quickstart 
+
+```python
+from ariadne import execute_request, make_executable_schema
+
+
+type_defs = """
+    schema {
+        query Query
+    }
+
+    type Query {
+        persons [Person]!
+    }
+
+    type Person {
+        firstName String
+        secondName String
```

Comment:
Here we have `secondName` field while in the resolver there is `lastName`. I think `lastName` fits better.

Reply:
I would say that l10n-wise the _correct_ way to do it is to have `givenName` and `familyName` 😉